config/opendatahub-io/org.yaml

```diff
@@ -279,6 +284,7 @@ orgs:
       repo-rest-proxy-write:
         description: "Members with Write access to the rest-proxy repository"
         members:
+        - amadhusu
```

Comment:
You don't need this access. I'm guessing you copied Humair's permissions, and his access here is an old artifact of a previous project he worked on.